zim/notebook/index/pages.py

```diff
@@ -539,6 +539,17 @@ def walk_bottomup(self, path=None):
 		page_id = self._pages.get_page_id(path) if path else ROOT_ID # can raise
 		return self._pages.walk_bottomup(page_id)
 
+	def search_pagename_substring(self, nameFragment):
+		candidates = self.db.execute(
+			'SELECT * FROM pages WHERE LOWER(name) LIKE ? ORDER BY parent,sortkey',
```

Comment:
I'll think about this too... for now, though, I'm just happy to have the productivity boost!